contributors/guide/README.md

```diff
@@ -210,6 +210,8 @@ To make it easier for your PR to receive reviews, consider the reviewers will ne
 * break large changes into a logical series of smaller patches which individually make easily understandable changes, and in aggregate solve a broader issue
 * label PRs with appropriate SIGs and reviewers: to do this read the messages the bot sends you to guide you through the PR process
 
+If your pull request isn't getting enough attention, use #pr-reviews to find reviewers!
```

Comment:
Maybe this should go at the end of the code-review section i.e. below the last para in this section...

Also, I think it'd be better if we said: `use the [#pr-reviews](https://kubernetes.slack.com/messages/pr-reviews) channel on Slack to get help finding reviewers`.